### PR TITLE
official/resnet: Clarify data format of pre-trained model.

### PR DESCRIPTION
**Pull request by asimshankar**
Motivated by #3418

### CODE REVIEW

**Comment by nealwu** [comment]
I believe users should be able to just run the same model code using the checkpoint as the `model_dir` and then add the flag `--data_format=channels_last`.

In fact I just tried this myself with the current master code and current [pre-trained model](https://github.com/tensorflow/models/tree/master/official/resnet#pre-trained-model), and I'm able to continue training just fine with `--data_format=channels_last`, reaching a train accuracy of 75% after a few hundred steps.

**Comment by tfboyd** [comment]
This is a general TensorFlow problem that also exists from TPU to GPU as with GPU to CPU, if you have a saved model and not the original code that was written to support both data formats you are kind of stuck. This is over a year old at this point so I am closing.  

One trick that might work and not related to this is training with NHWC which grappler will change to NCHW to get better perf but I think the saved model still ends up NHWC.  Of course if you wanted to use MKL you may want NCHW for CPU.  It seems like there needs to be a better story and that is beyond this example other than telling people to create model code that supports both formats.  :-(
